conf/messages.en

```diff
@@ -792,8 +792,8 @@ sharesDisposal.totalShares.error.decimal = Number of shares held after this disp
 sharesDisposal.totalShares.error.invalid.characters = Number of shares held after this disposal must be a number that is 0 or more, like 100
 sharesDisposal.totalShares.error.tooLarge = Number of shares held after this disposal must be 9,999,999 or fewer
 
-sharesDisposal.cya.title = Check your answers
-sharesDisposal.cya.heading = Check your answers
+sharesDisposal.cya.title = Check your answers: shares
```

Comment:
@charlotteothon should this be `Check your answers: disposal of shares`

Reply:
@kberec yes